src/sentry/static/sentry/app/components/tagDistributionMeter/index.jsx

```diff
@@ -116,96 +145,66 @@ export default class TagDistributionMeter extends React.Component {
 
     return this.renderSegments();
   }
-
-  render() {
-    const {title} = this.props;
-
-    return (
-      <DistributionGraph>
-        <Title>{title}</Title>
-        {this.renderTag()}
-      </DistributionGraph>
-    );
-  }
 }
 
-const DistributionGraph = styled('div')`
-  position: relative;
-  font-size: 13px;
-  margin-bottom: 10px;
-`;
-
-const Title = styled('div')`
-  position: relative;
-  font-size: 13px;
-  margin: 10px 0 8px;
-  font-weight: bold;
-  z-index: 5;
-  line-height: 1;
-`;
-
 const colors = [
```

Comment:
For variables which are hardcoded constants, you should use all caps 

https://github.com/airbnb/javascript#naming--uppercase

Reply:
DONE